pytorch_lightning/loops/base.py

```diff
@@ -61,6 +61,14 @@ def connect(self, trainer: 'pl.Trainer', *args: Any, **kwargs: Any) -> None:
         """Connects Loop with all the necessary things like connectors and accelerators."""
         self.trainer = proxy(trainer)
 
+    def on_skip(self) -> Optional[Any]:
+        """
+        The function to run when :attr:`run` should be skipped.
```

Comment:
```suggestion
        The function to run when :meth:`run` should be skipped, determined by the condition in :attr:`skip`.
```

Reply:
Note that you used ``` :attr:`run` ``` in other docstrings. That's another TODO for you 😂

Reply:
where did I do that?

Reply:
```python
pytorch_lightning/loops//base.py:29:        * :attr:`reset` (method): Resets the internal state between multiple calls of :attr:`run`
pytorch_lightning/loops//base.py:30:        * :attr:`advance` (method): Implements one step of the loop
pytorch_lightning/loops//base.py:68:        Will frequently check the :attr:`done` condition and calls :attr:`advance`
pytorch_lightning/loops//base.py:95:        """Resets the internal state of the loop at the beginning of each call to :attr:`run`."""
pytorch_lightning/loops//base.py:99:        Hook to be called as the first thing after entering :attr:`run` (except the state reset).
pytorch_lightning/loops//base.py:101:        Accepts all arguments passed to :attr:`run`.
pytorch_lightning/loops//base.py:107:        Hook to be called each time before :attr:`advance` is called. Accepts all arguments passed to :attr`run`.
pytorch_lightning/loops//base.py:113:        """Performs a single step. Accepts all arguments passed to :attr:`run`."""
pytorch_lightning/loops//base.py:116:        """Hook to be called each time after :attr:`advance` is called."""
pytorch_lightning/loops//base.py:119:        """Hook to be called at the end of the run. Its return argument is returned from :attr:`run`."""
```

Should all these be `:meth:`?

Reply:
yes, meth is for methods and attr for properties. They also need the class normally. 
but doesn't matter, docs are not visible yet.